.github/workflows/create-docker-prerelease.yml

```diff
@@ -0,0 +1,60 @@
+name: Create docker prerelease
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Create docker prerelease
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ secrets.DOCKER_ORG }}/${{ secrets.DOCKER_REPOSITORY }}
+
+      - name: Build and push Docker images
+        uses: docker/build-push-action@v4
+        id: build_push_docker
+        with:
+          context: .
+          platforms: linux/amd64
+          tags: ${{ steps.meta.outputs.tags }}
+          push: true
+
+      - name: Clean Docker image
+        run: docker rmi ${{ steps.build_push_docker.outputs.imageid }} -f
+
+      - name: Docker meta (CI)
+        id: meta-ci
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ secrets.DOCKER_ORG }}/${{ secrets.DOCKER_REPOSITORY }}
+          flavor: |
+            prefix=ci-,onlatest=true
```

Comment:
Same here, we might want something like:
I think we want to set a tag here. I think the following is the convention:
```
tags: |
  type=edge
flavor: |
  prefix=ci-
  latest=false
```